.github/workflows/website.yml

```diff
@@ -42,16 +42,14 @@ jobs:
           NEXT_PUBLIC_API: https://api-staging.web3.storage
           NEXT_PUBLIC_MAGIC: ${{ secrets.STAGING_MAGIC_PUBLIC_KEY }}
 
-      # TODO: replace with adding to web3.storage
-      # Pin the built site to ipfs-cluster, output the cid as `steps.ipfs.outputs.cid`
-      - name: Add to IPFS
-        uses: ipfs-shipyard/ipfs-github-action@v2.1.0
+      # Add the site to web3.storage, output the cid as `steps.ipfs.outputs.cid`
+      - name: Add to web3.storage
+        uses: web3-storage/add-to-web3@mvp
         id: ipfs
         with:
           path_to_add: packages/website/out
-          cluster_host: /dnsaddr/filecoin.storage.ipfscluster.io
-          cluster_user: ${{ secrets.CLUSTER_USER }}
-          cluster_password: ${{ secrets.CLUSTER_PASSWORD }}
```

Comment:
Just a reminder to remove these two secrets from the repo after merge as this is the only thing that was using them.

Reply:
IT IS DONE